src/cheevos/Cheevos.h

```diff
@@ -42,7 +43,13 @@ class CCheevos
                            const std::string& richPresence);
   void EnableRichPresence(const std::string& script);
   void EvaluateRichPresence(std::string& evaluation, unsigned int consoleID);
+  void ActivateAchievement(unsigned cheevo_id, const char* memaddr);
```

Comment:
Change the param name to `cheevoId` to match the code style. This applies for the other methods as well